env/frontend.env

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
+PORT=8062
+MITOPEN_AXIOS_WITH_CREDENTIALS=true
+WEBPACK_ANALYZE=True
```

Comment:
This causes my watch container to throw an error and exit

```
WEBPACK_ANALYZE: Invalid bool input: "True"
```

Reply:
> This causes my watch container to throw an error and exit
> 
> ```
> WEBPACK_ANALYZE: Invalid bool input: "True"
> ```

I get this same error

Reply:
@rhysyngsun I think this needs to be lowercase `true` because the var is evaluated by Javascript